### PR TITLE
fix: log acscsEmail errors to admin events

### PR DESCRIPTION
**Pull request by johannes94**
### Description

This PR adds logging to the acscsEmail integration so that error logs will appear in AdministrationEvents to allow customers of ACSCS who don't have access to central logs to see problems with their email sending.

This is important to communicate e.g rate limitting issues.

### User-facing documentation

(*must be* 2 items and both *must be* checked)

- [ ] CHANGELOG is updated
- [x] CHANGELOG update is not needed
- [ ] Documentation PR is created and linked above
- [x] Documentation is not needed

### Testing

- [ ] inspected CI results

#### Automated testing

(*must be* at least 1 item and all items *must be* checked)

- [ ] added unit tests
- [ ] added e2e tests
- [ ] added regression tests
- [ ] added compatibility tests
- [ ] modified existing tests
- [x] contributed **no automated tests**
  

#### How I validated my change

- Deploy a central to an openshift cluster use `ROX_MANAGED_CENTRAL=true`
- Create a acscsEmail integration
- Hit test on that integration
- Make sure an error log appears in adminstation events in UI

### CODE REVIEW

File: central/notifiers/acscsemail/acscsemail.go [hunk 1]
```diff
@@ -66,6 +73,7 @@ func (e *acscsEmail) AlertNotify(ctx context.Context, alert *storage.Alert) erro
 	subject := notifiers.SummaryForAlert(alert)
 	body, err := email.PlainTextAlert(alert, e.notifier.UiEndpoint, e.mitreStore)
 	if err != nil {
+		e.logError("generating email content failed", err)
```
Comment (kurlov) [inline central/notifiers/acscsemail/acscsemail.go:76]: nit: Capitalize first word. Feel free to ignore

**Review by kurlov — APPROVED** [review]
🎉

**Review by stehessel — APPROVED** [review]

**Comment by johannes94** [comment]
Failures are unrelated going to merge.

**Pull request merged**